lib/rubygems/deprecate.rb

```diff
@@ -70,7 +95,7 @@ def deprecate(name, replacement=:none)
   end
 
   # Deprecation method to deprecate Rubygems commands
-  def deprecate_command
```

Comment:
I think the names convey it quite well, but still we should make it super clear that this is an internal module, not a public one that can be used by anyone. Not sure which strategies are tipically used for this.